pkg/cmd/cli/cmd/loginoptions.go

```diff
@@ -372,3 +380,37 @@ func (o *LoginOptions) serverProvided() bool {
 	_, err := o.ClientConfig.ClientConfig()
 	return err == nil || !clientcmd.IsNoServerFound(err)
 }
+
+func relativizeClientConfigPaths(cfg *clientcmdapi.Config, base string) (err error) {
```

Comment:
Yeah, should really be upstream. You need to do this any time you write a kubeconfig to disk